tests/cephfs/cephfs_utilsV1.py

```diff
@@ -245,3 +248,29 @@ def client_clean_up(fuse_clients, kernel_clients, mounting_dir, *args, **kwargs)
                 client.exec_command(sudo=True, cmd="iptables -F", check_ec=False)
 
         return 0
+
+    def get_all_subvolumes(self, client, fs_list=[]):
+        """
+        it gets all the subvolume details in fs provided
+        if fs_list is empty it will iterate through all the fs present and gets back the list
+        Args:
+            client:
+            fs_list:
+
+        Returns:
+            list of all sub volumes present in the group
+        """
+        if fs_list == []:
+            out, rc = client.exec_command(
+                sudo=True, cmd="ceph fs ls --format json-pretty"
```

Comment:
Could we switch the format type to `json` instead of `json-pretty`? The current implementation would work in most cases however it could introduce indentations and carriage returns...